runtime/lua/vim/lsp/rpc.lua

```diff
@@ -325,7 +323,15 @@ local function start(cmd, cmd_args, dispatchers, extra_spawn_params)
     end
     handle, pid = uv.spawn(cmd, spawn_params, onexit)
     if handle == nil then
-      error(string.format("start `%s` failed: %s", cmd, pid))
+      local msg = string.format("Spawning language server with cmd: `%s` failed", cmd)
+      if string.match(pid, "ENOENT") then
```

Comment:
I sanitized the name here, but open to removing it

Reply:
Seems fine to me